keps/sig-instrumentation/647-apiserver-tracing/kep.yaml

```diff
@@ -22,10 +22,11 @@ prr-approvers:
 see-also:
 replaces:
 stage: alpha
-last-updated: 2021-07-15
```

Comment:
Can you please update stage to beta? That will require you to update PRR. I also see this is missing the beta PRR questionnaire.

LGTM from the SIG Instrumentation perspective.

Reply:
updated.   Can you be my PRR approver, or should I get someone else?